src/node-red/flowfuse/index.md

```diff
@@ -0,0 +1,20 @@
+---
+eleventyNavigation:
+  key: FlowFuse Nodes
+  order: 5
+meta:
+  title: FlowFuse Exlusive Nodes
+  description: A detailed overview of the Node-RED nodes that are exclusive to FlowFuse, including their purpose, usage, and benefits for building integrations and automations.
```

Comment:
detailed + overview is a bit self contradicting!  It is either an overview or detailed!

Also, lets be inclusive by default. We can call them "provided by FlowFuse" or "developed by FlowFuse" etc